spring-cloud-dataflow-server-core/src/main/resources/META-INF/dataflow-server-defaults.yml

```diff
@@ -117,3 +117,8 @@ spring:
             - GET    /tasks/executions/*             => hasRole('ROLE_VIEW')
             - POST   /tasks/executions               => hasRole('ROLE_CREATE')
             - DELETE /tasks/executions/*             => hasRole('ROLE_CREATE')
+
+            # Tools
+
+            - POST   /tools/**                       => hasRole('ROLE_CREATE')
```

Comment:
This line can be deleted, right? `/tools/**` covers all tools endpoints from the `ToolsController`.